de.dlr.sc.virsat.model.extension.requirements.feature/.settings/org.eclipse.core.resources.prefs

```diff
@@ -0,0 +1,2 @@
+eclipse.preferences.version=1

```

Comment:
There should also be mylyn.team.ui.prefs somewhere

Reply:
The mylyn prefs seem to be missing in the other projects as well.

Reply:
Yes mylyn preferences have to be placed in projects. Usually they can be copied from other projects.